src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/MvtReader.java

```diff
@@ -325,7 +335,11 @@ private static Geometry readLines(GeometryFactory geomFactory, List<Integer> geo
             geoms.add(geomFactory.createLineString(nextCoordSeq));
         }
 
-        return geoms.size() == 1 ? geoms.get(0) : geomFactory.createMultiLineString(geoms.toArray(new LineString[geoms.size()]));
+        if(geoms.size() == 1) {
```

Comment:
Same here, ternary operator should be supported?

Reply:
this was replaced (1) for the line length (2) Findbugs / checkstyle complain about complicated ternary statements.  I can revert 👍